topology/probes/istio/destinationrule.go

```diff
@@ -49,3 +52,24 @@ func (h *destinationRuleHandler) Dump(obj interface{}) string {
 func newDestinationRuleProbe(client interface{}, g *graph.Graph) k8s.Subprobe {
 	return k8s.NewResourceCache(client.(*kiali.IstioClient).GetIstioNetworkingApi(), &kiali.DestinationRule{}, "destinationrules", g, &destinationRuleHandler{})
 }
+
+type destinationRuleSpec struct {
+	App string `mapstructure:"host"`
+}
+
+func destinationRuleServiceAreLinked(a, b interface{}) bool {
+	dr := a.(*kiali.DestinationRule)
+	service := b.(*v1.Service)
+	drSpec := &destinationRuleSpec{}
+	if err := mapstructure.Decode(dr.Spec, drSpec); err != nil {
```

Comment:
Using mapstructure just to map a single attribute seems a bit overkill.

```
return dr.Spec["host"] == service.Labels["app"]
```
should be enough

Reply:
DONE